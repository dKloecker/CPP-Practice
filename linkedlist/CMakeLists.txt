# CMakeLists.txt

# Minimum CMake version required
cmake_minimum_required(VERSION 3.10)

# Project name and version
project(HelloWorld VERSION 1.0)

# Set C++ standard
set(CMAKE_CXX_STANDARD 20)
set(CMAKE_CXX_STANDARD_REQUIRED True)

# Add the executable
add_executable(hello_world 
  main.cpp
  linkedList.cpp
)

