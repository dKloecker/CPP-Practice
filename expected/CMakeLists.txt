cmake_minimum_required(VERSION 3.10)

project(expected VERSION 1.0)

set(CMAKE_CXX_STANDARD 20)
set(CMAKE_CXX_STANDARD_REQUIRED True)

add_executable(expected
  main.cpp
  expected.cpp
)
